test/lib/meetup_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+require_relative '../support/meetups/sample_api_responses'
+
+class MeetupTest < ActiveSupport::TestCase
+  test "some test" do
+    Meetup.stubs(:get).with("/pro/operationcode/groups", options).returns(group_endpoint_response)
+
+    parsed_response = Meetup.new.operationcode_data
+
+    assert_equal 2, parsed_response.size
```

Comment:
These are not the actual assertions you would make. 

They are just here to illustrate that the stubbed data is in fact being returned, and that you can work with it.